python/lsst/ci/build.py

```diff
@@ -329,14 +333,26 @@ def write_status(self):
 
     @staticmethod
     def run(args):
+        # Instantiate AgentManager and define agent
+        manager = AgentManager()
+        agent = manager.agent
+
         # Ensure build directory exists and is writable
         build_dir = args.build_dir
         if not os.access(build_dir, os.W_OK):
             raise Exception(f"Directory {build_dir!r} does not exist or isn't writable.")
 
+        # Load PR information saved by prepare.py
+        pr_info = Builder.load_pr_info(build_dir)
```

Comment:
Not sure why these are static methods on this class.  You might consider a class used to manage the PR info that's used by both `build.py` and `prepare.py`.